binder/environment.yml

```diff
@@ -18,3 +18,7 @@ dependencies:
   - hvplot
   - holoviews
   - panel
+
+  - pip
+  - pip:
+      - "."
```

Comment:
Will this install earthaccess from source in the Binder environment?

Reply:
Correct, but not editable. We can change it to `"--editable ."` to achieve that if we want. I haven't tested this -- is there a way to test with Binder before merge to `main`?